docs/workflows/input_output.md

```diff
@@ -16,63 +16,27 @@ The Cumulus Message Adapter and Cumulus Message Adapter libraries help task deve
 
-Once you've deployed the layer, include the CMA in the configured Lambda layers:
+Once you've deployed the layer, include the CMA layer in your  `cumulus_message_adapter_lambda_layer_arn` `tfvars` file if using the `cumulus` module for inclusion in all core Cumulus lambdas.
 
-```yaml
-    DiscoverPdrs:
-      layers:
-        - arn:aws:lambda:us-east-1:{{AWS_ACCOUNT_ID}}:layer:Cumulus_Message_Adapter:{version number}
-```
-
-In the future if you wish to update/change the CMA version you will need to update the deployed CMA, and update the layer configuration for the impacted Lambdas as needed, or re-run the Terraform module. If you have a large number of Lambdas utilizing the CMA, you can include a configuration key in your `config.yml`:
-
-```yaml
-    cma_layer: arn:aws:lambda:us-east-1:{{AWS_ACCOUNT_ID}}:layer:Cumulus_Message_Adapter:{version number}
-```
+Likewise, if including your own Lambda or ECS task Terraform modules, make sure to include that layer to include the CMA into your own code, and set a `CUMULUS_MESSAGE_ADAPTER_DIR` to `/opt` to make use of the CMA.
 
-and include the reference in the Lambda configuration:
-
-```yaml
-    DiscoverPdrs:
-      layers:
-        - '{{cma_layer}'
-```
+In the future if you wish to update/change the CMA version you will need to update the deployed CMA, and update the layer configuration for the impacted Lambdas as needed.}.
 
 ***Please Note***: Updating/removing a layer does not change a deployed Lambda, so to update the CMA you should deploy a new version of the CMA layer, update the associated Lambda configuration to reference the new CMA version, and re-deploy your Lambdas.
 
-***Please Note***: Updating the CMA without updating the lambda code will fail if operating with the Workflow Lambda Versions feature enabled.  If you are utilizing this option, we recommend continuing use of kes injection/manual addition for now.
-
-This method will be supported more fully once migration to Terraform Deployments has been completed.
-
 ### Manual Addition
 
-You can include the CMA package in the Lambda code in the `cumulus-message-adapter` sub-directory, this will achieve a similar result to kes injection, but will require manual updates to update the CMA code.
+You can include the CMA package in the Lambda code in the `cumulus-message-adapter` sub-directory in your lambda `.zip`, for any lambda runtime that includes a `python` [runtime](https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html).  `python 2` is included in lambda runtimes that use Amazon Linux, however `Amazon Linux 2` will not support this directly.
```

Comment:
```suggestion
You can include the CMA package in the Lambda code in the `cumulus-message-adapter` sub-directory in your lambda `.zip`, for any Lambda runtime that includes a `python` [runtime](https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html).  `python 2` is included in Lambda runtimes that use `Amazon Linux`, however `Amazon Linux 2` will not support this directly.
```

Also, should the first `python` in this message be `python 2`?

Reply:
No, we explicitly support 2.7, 3.6 for the CMA *for now*